parsera/parsera.py

```diff
@@ -34,25 +34,49 @@ def __init__(
         self.loader = PageLoader()
 
     async def _run(
-        self, url: str, elements: dict, proxy_settings: dict | None = None
+        self,
+        url: str,
+        elements: dict,
+        proxy_settings: dict,
+        scrolls_limit: int | None = None,
     ) -> dict:
-        content = await self.loader.load_content(url=url, proxy_settings=proxy_settings)
+        content = await self.loader.load_content(
+            url=url, proxy_settings=proxy_settings, scrolls_limit=scrolls_limit
+        )
         extractor_instance = self.extractor.value(
             elements=elements, model=self.model, content=content
         )
         result = await extractor_instance.run()
         return result
 
-    def run(self, url: str, elements: dict, proxy_settings: dict | None = None) -> dict:
+    def run(
+        self,
+        url: str,
+        elements: dict,
+        proxy_settings: dict,
+        scrolls_limit: int | None = None,
```

Comment:
Why is `scrolls_limit` `None` by default? When I call run with default it fails.